pkg/serialize/serialize.go

```diff
@@ -151,7 +157,7 @@ func (json *JSON) buildJSONPackage(p *spdx.Package) (jsonPackage v222.Package, e
 		CopyrightText:        p.CopyrightText,
 		HasFiles:             []string{},
 		Checksums:            []v222.Checksum{},
-		ExternalRefs:         []v222.ExternalRef{},
+		ExternalRefs:         externalRefs,
```

Comment:
@puerco any reason we did not add them in the initial implementation? Looks like we could do the same for the `Checksums`, right?

Reply:
No reason, I just missed adding those. Thanks for fixing it @sbs2001 !

Now, not all packages have checksums, and I see they seems to be there when expected can we file an issue if we are missing them somewhere?

For example:
![image](https://user-images.githubusercontent.com/3935899/179566643-f7bcf1a8-0283-4e63-b260-25b9cb1fa289.png)

Reply:
Probably we should add omit empty to non required fields too :thinking: